roles/edpm_deploy/tasks/main.yml

```diff
@@ -136,13 +136,15 @@
       ansible.builtin.debug:
         var: cifmw_edpm_deploy_crs_kustomize_result
 
-    - name: Apply the OpenStackDataPlaneNodeSet CR
-      when: not cifmw_edpm_deploy_dryrun
-      cifmw.general.ci_script:
-        output_dir: "{{ cifmw_edpm_deploy_basedir }}/artifacts"
-        script: "oc apply -f {{ cifmw_edpm_deploy_crs_kustomize_result.output_path }}"
+    - name: Apply dataplane resources but ignore DataPlaneDeployment
```

Comment:
```suggestion
    - name: Apply dataplane resources but ignore "{{ cifmw_edpm_deploy_step2_kind }}"
```

Reply:
I will change in a follow up PR, adding also baremetal, thanks